dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -358,9 +359,18 @@ def report(self, remove_disabled_flag: bool = False) -> dict:
 
         return profile
 
-    def _load_hist_helper(self, data):
-        # Pop off stuff specific to numerical stats
-        hist = data.pop("_stored_histogram")
+    def _load_stats_helper(self, deserialized_data: dict):
+        """Assistance function in the deserialization of profiler objects.
+
+            This function is to be used to enforce correct typing for attributes
+            associated with the NumericStatsMixinconversions when loading profiler
+            objects in from their serialized saved format
+        :param deserialized_data: The dictionary read in from the serialized form
+            of the profiler object
+        :type deserialized_data: Dict
+        """
+        # Pop off attributes specific to numerical stats histogram creation
+        hist = deserialized_data.pop("_stored_histogram")
```

Comment:
do we even need to pass in the `deserialized_data`? are the values already assigned and can we just update those instead?